### PR TITLE
#307 Make the story editor workspace endless on scroll to the right and bottom

### LINKED ISSUE #307: 🎨 📜 Make the story editor workspace endless on scroll to the right and bottom
## User stories

As a user I should be able to scroll on the editor frame endlessly to the right and towards the bottom. Currently am able to scroll to the right/bottom only when I add a block to that side. I should be able to do this whether there is a block or not.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Screenshot

Example:
When scrolling to the right, am only able to scroll as far as the "Great. Nice to finally meet you {{name}} 😃" Message Block
image

I should be able to create extra space when I scroll regardless of whethere there is a block there or not.

### Acceptance Criteria

1. I should be able to create extra space when I scroll regardless of whethere there is a block to that side or not.
2. Record a short video/gif to show how your implentation works.

Please also note:

1. Once this is implemented, you can remove the story editor scroll bar as it conflicts with the main page scroll bar

### PR DESCRIPTION
**Pull request by jobkarani**
# Description

This pull request makes the story editor workspace endless on scroll and bottom. To achieve this I had to install the **ngx-infinite-scroll** package.

Fixes #307

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
[Elewa Conversational Learning Manager (5).webm](https://user-images.githubusercontent.com/53146890/221153081-61922a73-8e16-4bcd-b5e6-6d7c1085714c.webm)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/convs-mgr/stories/editor/src/lib/components/editor-frame/editor-frame.component.ts [hunk 1]
```diff
@@ -17,10 +17,15 @@ export class StoryEditorFrameComponent implements AfterViewInit //implements OnD
...
 
+
   ngAfterViewInit() {
     const frame = this._frameInitialiser.initialiseEditor(this.editorVC, this.viewport);
 
     this.frameLoaded.emit(frame);
+  }
 
+  onScroll(event: any): void {
+    console.log(event)
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/editor/src/lib/components/editor-frame/editor-frame.component.ts:28]: Remove this console.log

File: libs/features/convs-mgr/stories/editor/src/lib/convl-story-editor.module.ts [hunk 2]
```diff
@@ -24,6 +24,8 @@ import { ManageChannelStoryLinkService } from './providers/manage-channel-story-
 import { AddBotToChannelModal } from './modals/add-bot-to-channel-modal/add-bot-to-channel.modal';
 
 import { ConvlStoryEditorRouterModule } from './convs-story-editor.router.module';
+import { InfiniteScrollModule } from 'ngx-infinite-scroll';
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/editor/src/lib/convl-story-editor.module.ts:27]: Check order of imports. External imports should come before internal imports. 'ngx-infinite-scroll' should come after angular imports and before internal imports

**Review by ReaganCn — CHANGES_REQUESTED** [review]
Good job. Just some few changes

**Comment by jobkarani** [comment]
@ReaganCn the requested changes have been implemented. Please review again.

**Review by ReaganCn — APPROVED** [review]

**Review by ReaganCn — APPROVED** [review]
Good work

**Pull request merged**